src/ngraph/builder/quantization.cpp

```diff
@@ -121,13 +121,20 @@ namespace ngraph
                                            const std::shared_ptr<Node> max_freezed_output,
                                            const bool with_relu)
         {
+            bool is_output_signed = !with_relu;
             float scale = builder::quantization_util::get_scale(min_input,
                                                                 max_input,
                                                                 min_filter,
                                                                 max_filter,
                                                                 min_freezed_output,
-                                                                max_freezed_output);
+                                                                max_freezed_output,
+                                                                is_output_signed);
```

Comment:
Instead of booleans, we should be passing element::Type and then getting the information we want from that Type in the function.  Booleans do not make great APIs.